flahub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
this is disappointing

Reply:
I'm still searching ways to build flutter for arm64 arch :sweat_smile: